core/src/main/java/com/linecorp/armeria/internal/common/AbstractHttp2ConnectionHandler.java

```diff
@@ -111,13 +112,24 @@ protected final void onConnectionError(ChannelHandlerContext ctx, boolean outbou
 
         handlingConnectionError = true;
         if (Exceptions.isExpected(cause) || isGoAwaySentException(cause, connection())) {
-            // Ignore silently.
+            if (logger.isDebugEnabled()) {
+                logger.debug("{} HTTP/2 connection error:", ctx.channel(), cause);
+            }
```

Comment:
```suggestion
            logger.debug("{} HTTP/2 connection error:", ctx.channel(), cause);
```
We use `isDebugEnabled` when it takes cost to create instances that are logged.

Reply:
Ah, I thought that there would be some cost if a throwable was passed in.
Went through the code and verified at least `logback` terminates early 👍